src/index.js

```diff
@@ -49,4 +30,62 @@ const listFooter = `</ul>
   <button type="button">Clear all completed</button>
 </div>`;
 
-document.querySelector('.content-1-1').innerHTML = listHead + data.map((item) => listItem(item)).join('') + listFooter;
+const display = () => {
+  document.querySelector('.content-1-1').innerHTML = listHead + data.map((item) => listItem(item)).join('') + listFooter;
+
+  document.querySelectorAll('.value').forEach((input, i) => {
+    input.onclick = (e) => {
+      const activeElement = document.querySelector('.active');
+
+      if (activeElement) {
+        const icons = [...activeElement.childNodes].filter((icon) => icon.innerHTML === 'more_vert' || icon.innerHTML === 'delete');
+
+        icons[0].classList.remove('hide');
+        icons[1].classList.add('hide');
+
+        activeElement.classList.remove('active');
+      }
+      e.target.parentElement.parentElement.classList.add('active');
+      const icons = [...e.target.parentElement.parentElement.childNodes].filter((icon) => icon.innerHTML === 'more_vert' || icon.innerHTML === 'delete');
+      icons[0].classList.add('hide');
+      icons[1].classList.remove('hide');
+
+      icons[1].onclick = () => {
+        if (!input.value) {
+          data = removeTask(i, data);
+
+          display();
+        }
+      };
+
+      e.target.onkeypress = (event) => {
+        if (event.key === 'Enter') {
+          e.target.parentElement.parentElement.classList.remove('active');
+          document.activeElement.blur();
+          icons[0].classList.remove('hide');
+          icons[1].classList.add('hide');
+
+          editTask(input, i, data);
+        }
+      };
+    };
+  });
+  const enter = document.querySelector('.enter');
+  enter.onkeypress = (e) => {
+    if (e.key === 'Enter') {
+      if (!enter.value) return;
+
+      addNewTask({
+        description: enter.value,
+        completed: false,
+        index: data.length,
```

Comment:
- [ ] [OPTIONAL] According to the project requirement by default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5) . Please fix it to make your project works as expected 👍